src/lib.rs

```diff
@@ -60,10 +60,14 @@
 #[macro_use]
 extern crate serde;
 
+#[cfg(feature = "mint")]
+extern crate mint;
```

Comment:
Please make it `pub extern crate mint;`. exposing the mint dependency allows a crate that already depends on euclid to just use euclid's mint and not worry about maintaining the versions in sync in Cargo.toml.

Also, it looks like this `extern crate mint;` is duplicated a few lines below.